source/reference/access-mdb-shell-help.txt

```diff
@@ -308,58 +363,27 @@ certain database commands and obtain information on your deployment:
    * - Help Methods and Commands
      - Description
 
-   * - ``help``
-
-     - Show help.
-
    * - ``db.help()``
-
      - Show help for database methods.
 
    * - ``db.<collection>.help()``
-
      - Show help on collection methods. The ``<collection>`` can be the
        name of an existing collection or a non-existing collection.
 
-   * - ``show dbs``
-
-     - Print a list of all databases on the server.
-
-       .. note::
-
-          ``show dbs`` is synonymous with ``show databases``.
-
-   * - ``use <db>``
-
-     - Switch current database to ``<db>``. The ``mongosh`` variable
-       ``db`` is set to the current database.
+   * - ``help``
+     - Show help.
 
    * - ``show collections``
-
      - Print a list of all collections for current database
 
-   * - ``show users``
-
-     - Print a list of users for current database.
-
-   * - ``show roles``
-
-     - Print a list of all roles, both user-defined and built-in, for
-       the current database.
-
-   * - ``show profile``
-
-     - Print the five most recent operations that took 1 millisecond or
-       more. See documentation on the
-       :manual:`database profiler
-       </tutorial/manage-the-database-profiler>` for more information.
+   * - ``show dbs``
+     - Print a list of all databases on the server.
```

Comment:
I would not use "Print". I would use "Display" or "Returns".

Same comment throughout.

Reply:
Done